relay/nakama/main.go

```diff
@@ -39,6 +39,12 @@ const (
 )
 
 type receiptChan chan *Receipt
+type DevMode int
+
+const (
+	Debug DevMode = iota
```

Comment:
what are your thoughts on if we just simplify this to `DebugEnabled bool`?

Reply:
idc, I did it this way because it's a bit more bijective with an environment variable. bool only has two possibilities but an environment variable can be any string.

Reply:
i think as a dev id prefer something like:

`export ENABLE_DEBUG=true`

if the env is enitrely unset, we dont use debug

Reply:
sure